### PR TITLE
NT Vote Estimation

### PR DESCRIPTION
**Pull request by noumantahir**
updated estimate vote value calculation recipe to match with Hive recipe

Screenshot 2021-07-13 at 1 31 49 PM
Screenshot 2021-07-13 at 1 37 22 PM

### CODE REVIEW

**Review by feruzm — APPROVED** [review]
Lgtm

File: src/utils/conversions.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+};
+
+export const vestsToRshares = (vests:number, votingPower:number, weight:number) => {
+  if (!vests || !votingPower || !weight) {
+    return 0;
+  }
+
+  const finalVest = vests * 1e6;
+  const power = (votingPower * weight / 10000) / 50
+  return (power * finalVest / 1000)
```
Comment (feruzm) [inline src/utils/conversions.ts:16]: @noumantahir here one 0 missing, no? used to be `/1e4`

Comment (noumantahir) [reply src/utils/conversions.ts:16]: Will compare shortly

Comment (noumantahir) [reply src/utils/conversions.ts:16]: I double checked, only thing I could find varying is weight, as I am used weight calculation from previous formula as not information given in their documentation.

`  const weight = sliderValue * 10000;`

**Pull request merged**